lib/trino-parquet/src/test/java/io/trino/parquet/reader/TestingColumnReader.java

```diff
@@ -491,6 +491,13 @@ private static Assertion<Number> assertTime(int precision, int rounding)
         };
     }
 
+    private static Assertion<Number> assertTimeInt(int precision)
```

Comment:
what is "Int-ish" about this method ("...Int" in the name)?

why `assertTime` doesn't work for the needs here?

Reply:
it's not needed, I was able to use `assertTime` directly